master/internal/db/postgres_jobs.go

```diff
@@ -35,14 +38,14 @@ VALUES (:job_id, :job_type, :owner_id, :q_position)
 	return nil
 }
 
-// UpdateJob propagates the new queue position to the job.
-func (db *PgDB) UpdateJob(job *model.Job) error {
-	if job.JobID.String() == "" {
+// UpdateJobPosition propagates the new queue position to the job.
+func (db *PgDB) UpdateJobPosition(jobID fmt.Stringer, position decimal.Decimal) error {
```

Comment:
`fmt.Stringer` -> `model.JobID` no matter what `interfacer` said lol (maybe you called `.String()` but that wouldn't be needed)